client/package.json

```diff
@@ -121,17 +117,17 @@
     "@types/lodash.sortby": "^4.7.6",
     "@types/lodash.uniq": "^4.5.6",
     "@types/lodash.zip": "^4.2.6",
+    "@types/node": "^20.10.6",
```

Comment:
DP is using Node 20.10.6 starting this PR - do we need to update Explorer side too? https://github.com/chanzuckerberg/single-cell-data-portal/pull/6431